### DISCUSSION TITLE
How to combine LaTeX and columns in quarto presentation?

### DISCUSSION THREAD

**Discussion by uriahf** [quarto-dev/quarto-cli#4425] [Q&A] [answered]
I'm trying to combine LaTeX and columns in my quarto presentation, but it does not work as expected:

```
---
title: "Net Benefit"
format: revealjs
editor: visual
---

## Net Benefit

:::: {.columns}

::: {.column }
$$\begin{aligned}
\text{Net Benefit} = \frac{\text{TP}}{\text{N}} - \frac{\text{FP}}{\text{N}} * {\frac{{p_{t}}}{{1 - p_{t}}}}
\end{aligned}$$
:::

::: {.column }
I love NB
:::

::::
```

![image](https://user-images.githubusercontent.com/11351434/219942254-2fd6b28e-0e1f-4975-9e7f-f3cc5f41e2ab.png)

**Comment by mcanouil** [comment, marked as answer]
The column layout you are using is 50/50, thus it works as expected.
Your equation is simply overflowing on the second column.

<img width="1468" alt="image" src="https://user-images.githubusercontent.com/8896044/219943372-d9be0132-315b-4436-b85c-3a6f34038329.png">

```qmd
---
title: "Net Benefit"
format: revealjs
---

## Net Benefit

:::: {.columns}

::: {.column style="font-size: 50%;"}
$$
\begin{aligned}
\text{Net Benefit} = \frac{\text{TP}}{\text{N}} - \frac{\text{FP}}{\text{N}} * {\frac{{p_{t}}}{{1 - p_{t}}}}
\end{aligned}
$$
:::

::: {.column }
I love NB
:::
```

**Reply by uriahf** [reply]
Thank you so much!
